api/type.rb

```diff
@@ -325,46 +325,38 @@ def field_name
 
     # Represents a reference to another resource
     class ResourceRef < Type
-      ALLOWED_WITHOUT_PROPERTY = [SelfLink::EXPORT_KEY].freeze
-
-      attr_reader :resource
-      attr_reader :imports
-
-      def out_type
-        resource_ref.out_name
-      end
+      attr_reader :resources
```

Comment:
I feel like this should be called `resource_refs`. Most of the places in this code right now are referring to `property.resources` which could end up being confusing to people, since we already know that resources have properties but apparently these properties also have resources.

Reply:
Agree.  `resource_types` would also work for me, if you prefer not to further overload `resource_refs` as a term.

Reply:
`resource_refs` sounds like a good idea. I'll make that change.